utoipa-swagger-ui/src/axum.rs

```diff
@@ -38,19 +39,24 @@ where
         } else {
             Config::new(urls)
         };
+        let config = Arc::new(config);
 
         router.route(
-            swagger_ui.path.as_ref(),
-            routing::get(serve_swagger_ui).layer(Extension(Arc::new(config))),
+            &swagger_ui.path,
+            routing::get(serve_swagger_ui).layer(Extension(config.clone())),
+        ).route(
+            &format!("{}*tail", swagger_ui.path),
+            routing::get(serve_swagger_ui).layer(Extension(config.clone())),
```

Comment:
No need to `.clone()` here since it is the last usage, thus it can be moved.
```suggestion
            routing::get(serve_swagger_ui).layer(Extension(config)),
```